### PR TITLE
Documentation Workflow Fixes

### PR DESCRIPTION
**Pull request by mikewiebe**
This PR fixes a few issues with the workflow to setup a working environment and minor refactoring of some of the phrasing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
 ### Step 1 - Installing the example repository
 
-To simplify the usage of the collection we are providing you with an [example repository](https://github.com/netascode/ansible-dc-vxlan-example) that you can clone from github which creates the proper skeleton required, including examples for pipelines. To clone the repository requires the installation of [git client](https://git-scm.com/downloads) that is available for all platforms.
+To simplify the usage of the collection we are providing you with an [example repository](https://github.com/netascode/ansible-dc-vxlan-example) that you can clone from github which creates the proper skeleton required, including examples for pipelines. Cloaning this repository requires the installation of [git client](https://git-scm.com/downloads) that is available for all platforms.
```
Comment (mtarking) [inline README.md:11]: Capitalize GitHub and update sentence to maybe something like this:

```that you can clone from GitHub which creates the proper skeleton structure required```

File: README.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ This will clone the repository into the directory nac-vxlan.
 
 ### Step 2 - Create the virtual environment with pyenv
 
-In this directory you will now create the new virtual environment. For pyenv to work you have to install a version of Python that you want to utilize. At the _time of this writting_, a common version utilized is python version 3.10.13 so to install this with pyenv would be the command `pyenv install 3.10.13`. For detailed instructions please visit the [pyenv](https://github.com/pyenv/pyenv) site.
+In this directory you will now create the new virtual environment and install a python version of your choice. At the _time of this writting_, a commonly used version is python version 3.10.13.  Command `pyenv install 3.10.13` will install this version. For detailed instructions please visit the [pyenv](https://github.com/pyenv/pyenv) site.
```
Comment (mtarking) [inline README.md:23]: Capitalize Python in two places

File: README.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
 # Ansible NDFC VXLAN Example Repository
 
-This repository is designed to build the skeleton for the Network as Code DC VXLAN Ansible Galaxy collection. Utilizing this repository will create a working structure that you can build upon to automate your Cisco Nexus Data Center network using NDFC.
+This repository is designed to build the skeleton for the Network as Code DC VXLAN Ansible Galaxy collection. Cloaning this repository will create a working structure that you can build upon to automate your Cisco Nexus Data Center network using NDFC.
```
Comment (mtarking) [inline README.md:3]: Should Collection be capitalized?

Comment (mtarking) [inline README.md:3]: ...using Nexus Dashboard Fabric Controller (NDFC).

File: README.md [hunk 4]
```diff
@@ -57,7 +57,7 @@ ansible-galaxy collection install -p collections/ansible_collections/ -r require
 
 You will need to then configure your ansible.cfg file to point to the correct location of the collection. 
 
-Which is going to tell you the path for all the python modules and libraries of the virtual environment that was created. If you look in that directory you will find the collections package locations. Here is the base ansible.cfg, you will need to adjust the collection_path to your environment paths:
+This sets the correct path for all the python modules and libraries in the virtual environment that was created. If you look in that directory you will find the collections package locations. Here is the base ansible.cfg, you will need to adjust the collection_path to your environment paths:
```
Comment (mtarking) [inline README.md:60]: Caps Python

Comment (mtarking) [inline README.md:60]: This should be ```collections_path```

**Review by mtarking — CHANGES_REQUESTED** [review]

**Review by rafmuller — APPROVED** [review]

**Review by rafmuller — APPROVED** [review]

**Review by rafmuller — APPROVED** [review]

**Review by mtarking — APPROVED** [review]

**Pull request merged**